toxic/provision.sh

```diff
@@ -43,6 +32,7 @@ apt-get -y install python3.5
 apt-get -y install python3.6
 apt-get -y install python3.7
 apt-get -y install python3.8
+apt-get -y install python3.9
 apt-get -y install python3-pip
 pip3 install tox
```

Comment:
```suggestion
pip3 install tox nox
```

Maybe we can add nox while we're at it? It is needed for PyUAVCAN/Yakut.

Reply:
what's wrong with tox? Why are we using two different but similar things?

Reply:
Tox is a bit too limited for PyUAVCAN and Yukon. In order to run the full suite of integration tests you need to manipulate the environment extensively, and I found that Tox is not an adequate tool for the job. It would work well for PyDSDL, I suppose, but not for these two.